stories/topics/proc-gcp-delete-extension-nodes.adoc

```diff
@@ -0,0 +1,25 @@
+[id="proc-gcp-delete-extension-nodes"]
+
+= Deleting Extension Nodes
+
+[IMPORTANT]
+====
+This action cannot be reversed.
+====
+
+Extension nodes are created as deployments in GCP Deployment Mananger. As such, the process to delete an Ansible Automation Platform extension node is the same as the process to delete the main Ansible Automation Platform deployment.  
+
+.Procedure
+. Select the main menu from the top left.
+. Select *Deployment Manager*.
+If you do not see *Deployment Manager*, select *View All Products*.
+. Tick the checkbox in front of the extension node you want to delete.
```

Comment:
If the checkbox has a name it should be used here.

Reply:
Check box does not have a name unfortunately